### PR TITLE
[JENKINS-55292] - Remove MailCommand localization from the core

### PR DESCRIPTION
**Pull request by oleg-nenashev**
See [JENKINS-55292](https://issues.jenkins-ci.org/browse/JENKINS-55292), the plugin was detached a while ago. It is a counterpart of https://github.com/jenkinsci/mailer-plugin/pull/48 

### Proposed changelog entries

* Remove MailCommand localization from the core
* ...

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@jenkinsci/core

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
On hold until https://github.com/jenkinsci/mailer-plugin/pull/48 is released

**Review by olivergondza — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Mailer plugin patch was released in 1.23 (Dec 2018. I believe we can go forward here

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 1]
```diff
@@ -21,32 +21,31 @@
 # THE SOFTWARE.
 
 InstallPluginCommand.InstallingPluginFromUrl=Instalando un plugin desde {0}
-InstallPluginCommand.NotAValidSourceName={0} no es un fichero/url v�lido, ni existe en el centro de actualizaciones 
+InstallPluginCommand.NotAValidSourceName={0} no es un fichero/url v�lido, ni existe en el centro de actualizaciones
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
InstallPluginCommand.NotAValidSourceName={0} no es un fichero/url válido, ni existe en el centro de actualizaciones 
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 2]
```diff
@@ -21,32 +21,31 @@
 # THE SOFTWARE.
 
 InstallPluginCommand.InstallingPluginFromUrl=Instalando un plugin desde {0}
-InstallPluginCommand.NotAValidSourceName={0} no es un fichero/url v�lido, ni existe en el centro de actualizaciones 
+InstallPluginCommand.NotAValidSourceName={0} no es un fichero/url v�lido, ni existe en el centro de actualizaciones 
 InstallPluginCommand.InstallingFromUpdateCenter=Instalando {0} desde el centro de actualizaciones
 InstallPluginCommand.InstallingPluginFromLocalFile=Instalando un plugin desde un fichero local: {0}
-InstallPluginCommand.NoUpdateDataRetrieved=No se recogi� todav�a informaci�n de actualizacion desde: {0}
+InstallPluginCommand.NoUpdateDataRetrieved=No se recogi� todav�a informaci�n de actualizacion desde: {0}
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
InstallPluginCommand.NoUpdateDataRetrieved=No se recogió todavía información de actualizacion desde: {0}
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 3]
```diff
@@ -21,32 +21,31 @@
...
 InstallPluginCommand.NoUpdateCenterDefined=No se ha definido un centro de actualizaciones en este Jenkins
-InstallPluginCommand.DidYouMean={0} parece un nombre de plugin corto. �Quieres decir ''{1}''?
-GetJobCommand.ShortDescription=Volcar el fichero XML de la definici�n de tarea a la salida est�ndard
+InstallPluginCommand.DidYouMean={0} parece un nombre de plugin corto. �Quieres decir ''{1}''?
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
InstallPluginCommand.DidYouMean={0} parece un nombre de plugin corto. ¿Quieres decir ''{1}''?
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 4]
```diff
@@ -21,32 +21,31 @@
...
+GetJobCommand.ShortDescription=Volcar el fichero XML de la definici�n de tarea a la salida est�ndard
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
GetJobCommand.ShortDescription=Volcar el fichero XML de la definición de tarea a la salida estándard
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 5]
```diff
@@ -21,32 +21,31 @@
...
 CopyJobCommand.ShortDescription=Copia una tarea
-InstallToolCommand.ShortDescription=Autom�ticamente instala componentes e imprime su localizaci�n a la salida est�ndard. S�lo se puede llamar desde un trabajo.
+InstallToolCommand.ShortDescription=Autom�ticamente instala componentes e imprime su localizaci�n a la salida est�ndard. S�lo se puede llamar desde un trabajo.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
InstallToolCommand.ShortDescription=Automáticamente instala componentes e imprime su localización a la salida estándard. Sólo se puede llamar desde un trabajo.
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 6]
```diff
@@ -21,32 +21,31 @@
...
 DeleteBuildsCommand.ShortDescription=Permite borrar registros
 InstallPluginCommand.ShortDescription=Instala un plugin desde un fichero, una URL o desde el centro de actualizaciones.
-VersionCommand.ShortDescription=Muestra la versi�n actual.
+VersionCommand.ShortDescription=Muestra la versi�n actual.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
VersionCommand.ShortDescription=Muestra la versión actual.
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 7]
```diff
@@ -21,32 +21,31 @@
...
 ListChangesCommand.ShortDescription=Vuelca el registro de cambios del trabajo actual.
-MailCommand.ShortDescription=Lee de la entrada est�ndard, y lo env�a por email.
-CreateJobCommand.ShortDescription=Permite crear una nueva tarea leyendo un fichero XML por la entrada est�ndard.
+CreateJobCommand.ShortDescription=Permite crear una nueva tarea leyendo un fichero XML por la entrada est�ndard.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
CreateJobCommand.ShortDescription=Permite crear una nueva tarea leyendo un fichero XML por la entrada estándard.
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 8]
```diff
@@ -21,32 +21,31 @@
...
 LoginCommand.ShortDescription=Guarda las credenciales para permitir que se puedan ejecutar comandos sin expecificar nuevamente las credenciales.
-SetBuildResultCommand.ShortDescription=Establece el resultado del trabajo actual. S�lo se puede ejecutar desde un trabajo.
+SetBuildResultCommand.ShortDescription=Establece el resultado del trabajo actual. S�lo se puede ejecutar desde un trabajo.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
SetBuildResultCommand.ShortDescription=Establece el resultado del trabajo actual. Sólo se puede ejecutar desde un trabajo.
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 9]
```diff
@@ -21,32 +21,31 @@
...
 GroovyCommand.ShortDescription=Ejecuta el script de groovy especificado.
 SetBuildDisplayNameCommand.ShortDescription=Establece el nombre a mostrar de un trabajo.
 BuildCommand.ShortDescription=Ejecuta una tarea, y opcionalmente espera hasta que acabe.
 HelpCommand.ShortDescription=Muestra la lista de todos los comandos disponibles.
 LogoutCommand.ShortDescription=Borra la credencial almacenada en el comando de login
 WhoAmICommand.ShortDescription=Muestra tus credenciales y permisos
-UpdateJobCommand.ShortDescription=Actualiza el fichero XML de la definici�n de una tarea desde la entrada est�ndard. Es lo contrario al comando get-job.
+UpdateJobCommand.ShortDescription=Actualiza el fichero XML de la definici�n de una tarea desde la entrada est�ndard. Es lo contrario al comando get-job.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
UpdateJobCommand.ShortDescription=Actualiza el fichero XML de la definición de una tarea desde la entrada estándard. Es lo contrario al comando get-job.
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 10]
```diff
@@ -21,32 +21,31 @@
...
 GroovyshCommand.ShortDescription=Ejecuta una shell interactiva de groovy.
-SetBuildDescriptionCommand.ShortDescription=Establece la descripci�n de una ejecuci�n.
+SetBuildDescriptionCommand.ShortDescription=Establece la descripci�n de una ejecuci�n.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
SetBuildDescriptionCommand.ShortDescription=Establece la descripción de una ejecución.
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 11]
```diff
@@ -57,11 +56,11 @@ DisconnectNodeCommand.ShortDescription=Desconectarse de un nodo
 QuietDownCommand.ShortDescription=Poner Jenkins en modo quieto y estar preparado para un reinicio. No comenzar ninguna ejecuci\u00f3n.
 CancelQuietDownCommand.ShortDescription=Cancelar el efecto del comando "quiet-down".
 OfflineNodeCommand.ShortDescription=Dejar de utilizar un nodo temporalmente hasta que se ejecute el comando "online-node".
-WaitNodeOnlineCommand.ShortDescription=Esperando hasta que el nodo est� activado
-WaitNodeOfflineCommand.ShortDescription=Esperando a que el nodo est� desactivado
+WaitNodeOnlineCommand.ShortDescription=Esperando hasta que el nodo est� activado
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
WaitNodeOnlineCommand.ShortDescription=Esperando hasta que el nodo esté activado
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 12]
```diff
@@ -57,11 +56,11 @@ DisconnectNodeCommand.ShortDescription=Desconectarse de un nodo
...
+WaitNodeOfflineCommand.ShortDescription=Esperando a que el nodo est� desactivado
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
WaitNodeOfflineCommand.ShortDescription=Esperando a que el nodo esté desactivado
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 13]
```diff
@@ -57,11 +56,11 @@ DisconnectNodeCommand.ShortDescription=Desconectarse de un nodo
...
 DisablePluginCommand.ShortDescription=\
-Deshabilita uno o m�s plugins instalados.
+Deshabilita uno o m�s plugins instalados.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
Deshabilita uno o más plugins instalados.
```

File: core/src/main/resources/hudson/cli/Messages_es.properties [hunk 14]
```diff
@@ -57,11 +56,11 @@ DisconnectNodeCommand.ShortDescription=Desconectarse de un nodo
...
 DisablePluginCommand.NoSuchStrategy=Esta estrategia ({0}) no existe. Las estrategias permitidas son {1}
 DisablePluginCommand.PrintUsageSummary=\
-Deshabilita los plugins con los nombre cortos dados. Puede definir c�mo proceder con los plugins dependientes y si se realiza un reinicio posterior.
+Deshabilita los plugins con los nombre cortos dados. Puede definir c�mo proceder con los plugins dependientes y si se realiza un reinicio posterior.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_es.properties]:
```suggestion
Deshabilita los plugins con los nombre cortos dados. Puede definir cómo proceder con los plugins dependientes y si se realiza un reinicio posterior.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 15]
```diff
@@ -3,9 +3,9 @@ InstallPluginCommand.InstallingFromUpdateCenter=Installazione di {0} dal Centro
 InstallPluginCommand.InstallingPluginFromLocalFile=Installazione di un plugin da un file locale in corso: {0}
 InstallPluginCommand.InstallingPluginFromStdin=Installazione di un plugin dallo standard input in corso
 InstallPluginCommand.InstallingPluginFromUrl=Installazione di un plugin da {0} in corso
-InstallPluginCommand.NoUpdateCenterDefined=Si noti che in quest''istanza di Jenkins non � definito alcun Centro aggiornamenti.
+InstallPluginCommand.NoUpdateCenterDefined=Si noti che in quest''istanza di Jenkins non � definito alcun Centro aggiornamenti.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
InstallPluginCommand.NoUpdateCenterDefined=Si noti che in quest''istanza di Jenkins non è definito alcun Centro aggiornamenti.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 16]
```diff
@@ -3,9 +3,9 @@ InstallPluginCommand.InstallingFromUpdateCenter=Installazione di {0} dal Centro
...
 InstallPluginCommand.NoUpdateDataRetrieved=Non sono ancora stati recuperati dati del Centro aggiornamenti da {0}
-InstallPluginCommand.NotAValidSourceName={0} non � n� un file valido, n� un URL, n� il nome di un artefatto di un plugin nel Centro aggiornamenti
+InstallPluginCommand.NotAValidSourceName={0} non � n� un file valido, n� un URL, n� il nome di un artefatto di un plugin nel Centro aggiornamenti
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
InstallPluginCommand.NotAValidSourceName={0} non è né un file valido, né un URL, né il nome di un artefatto di un plugin nel Centro aggiornamenti
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 17]
```diff
@@ -14,33 +14,32 @@ CopyJobCommand.ShortDescription=Copia un processo.
 CreateJobCommand.ShortDescription=Crea un nuovo processo leggendo lo standard input e interpretandolo come un file di configurazione XML.
 CreateNodeCommand.ShortDescription=Crea un nuovo nodo leggendo lo standard input e interpretandolo come un file di configurazione XML.
 CreateViewCommand.ShortDescription=Crea una nuova vista leggendo lo standard input e interpretandolo come un file di configurazione XML.
-DeleteBuildsCommand.ShortDescription=Elimina uno o pi� record relativi alle build.
-DeleteViewCommand.ShortDescription=Elimina una o pi� viste.
-DeleteJobCommand.ShortDescription=Elimina uno o pi� processi.
+DeleteBuildsCommand.ShortDescription=Elimina uno o pi� record relativi alle build.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
DeleteBuildsCommand.ShortDescription=Elimina uno o più record relativi alle build.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 18]
```diff
@@ -14,33 +14,32 @@ CopyJobCommand.ShortDescription=Copia un processo.
...
+DeleteViewCommand.ShortDescription=Elimina una o pi� viste.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
DeleteViewCommand.ShortDescription=Elimina una o più viste.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 19]
```diff
@@ -14,33 +14,32 @@ CopyJobCommand.ShortDescription=Copia un processo.
...
+DeleteJobCommand.ShortDescription=Elimina uno o pi� processi.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
DeleteJobCommand.ShortDescription=Elimina uno o più processi.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 20]
```diff
@@ -14,33 +14,32 @@ CopyJobCommand.ShortDescription=Copia un processo.
...
 InstallPluginCommand.ShortDescription=Installa un plugin da un file, un URL o dal Centro aggiornamenti.
-InstallToolCommand.ShortDescription=Esegue un''installazione automatica dello strumento e stampa il suo percorso sullo standard output. Pu� essere invocato solo da una build. [deprecato]
+InstallToolCommand.ShortDescription=Esegue un''installazione automatica dello strumento e stampa il suo percorso sullo standard output. Pu� essere invocato solo da una build. [deprecato]
 ListChangesCommand.ShortDescription=Esegue il dump del log delle modifiche per la/le build specificata/e.
 ListJobsCommand.ShortDescription=Elenca tutti i processi in una specifica vista o in un gruppo di elementi.
 ListPluginsCommand.ShortDescription=Visualizza un elenco di plugin installati.
 LoginCommand.ShortDescription=Salva le credenziali correnti per consentire ai comandi futuri di essere eseguiti senza specificare esplicitamente le informazioni relative alle credenziali. [deprecato]
-LoginCommand.FullDescription=A seconda dell''area di sicurezza sar� necessario fornire argomenti come:\n\
+LoginCommand.FullDescription=A seconda dell''area di sicurezza sar� necessario fornire argomenti come:\n\
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
LoginCommand.FullDescription=A seconda dell''area di sicurezza sarà necessario fornire argomenti come:\n\
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 21]
```diff
@@ -14,33 +14,32 @@ CopyJobCommand.ShortDescription=Copia un processo.
...
 --username UTENTE [ --password PASSWORD | --password-file FILE ]\n\
-Pu� non essere supportato in alcune aree di sicurezza come il single sign on.\n\
+Pu� non essere supportato in alcune aree di sicurezza come il single sign on.\n\
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
Può non essere supportato in alcune aree di sicurezza come il single sign on.\n\
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 22]
```diff
@@ -14,33 +14,32 @@ CopyJobCommand.ShortDescription=Copia un processo.
...
 Va abbinato al comando logout.\n\
 Le medesime opzioni possono essere utilizzate con qualsiasi altro comando, ma,\n\
 a differenza delle altre opzioni generiche dell''interfaccia da riga di comando,\n\
 queste devono essere specificate *dopo* il nome del comando.\n\
 Indipendentemente dal fatto che le credenziali siano salvate o meno, questa\n\
-modalit� di autenticazione non consentir� di fare riferimento (ad esempio)\n\
+modalit� di autenticazione non consentir� di fare riferimento (ad esempio)\n\
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
modalità di autenticazione non consentirà di fare riferimento (ad esempio)\n\
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 23]
```diff
@@ -51,14 +50,14 @@ GetNodeCommand.ShortDescription=Esegue il dump dell''XML della definizione del n
 GetViewCommand.ShortDescription=Esegue il dump dell''XML della definizione della vista sullo standard output.
 SetBuildDisplayNameCommand.ShortDescription=Imposta il nome visualizzato di una build.
 WhoAmICommand.ShortDescription=Visualizza i propri credenziali e permessi.
-UpdateJobCommand.ShortDescription=Aggiorna l''XML della definizione del processo dallo standard input. � il comando contrario di get-job.
-UpdateNodeCommand.ShortDescription=Aggiorna l''XML della definizione del nodo dallo standard input. � il comando contrario di get-node.
+UpdateJobCommand.ShortDescription=Aggiorna l''XML della definizione del processo dallo standard input. � il comando contrario di get-job.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
UpdateJobCommand.ShortDescription=Aggiorna l''XML della definizione del processo dallo standard input. È il comando contrario di get-job.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 24]
```diff
@@ -51,14 +50,14 @@ GetNodeCommand.ShortDescription=Esegue il dump dell''XML della definizione del n
...
+UpdateNodeCommand.ShortDescription=Aggiorna l''XML della definizione del nodo dallo standard input. � il comando contrario di get-node.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
UpdateNodeCommand.ShortDescription=Aggiorna l''XML della definizione del nodo dallo standard input. È il comando contrario di get-node.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 25]
```diff
@@ -51,14 +50,14 @@ GetNodeCommand.ShortDescription=Esegue il dump dell''XML della definizione del n
...
 SessionIdCommand.ShortDescription=Visualizza l''ID di sessione che cambia a ogni riavvio di Jenkins.
-UpdateViewCommand.ShortDescription=Aggiorna l''XML della definizione della vista dallo standard input. � il comando contrario di get-view.
+UpdateViewCommand.ShortDescription=Aggiorna l''XML della definizione della vista dallo standard input. � il comando contrario di get-view.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
UpdateViewCommand.ShortDescription=Aggiorna l''XML della definizione della vista dallo standard input. È il comando contrario di get-view.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 26]
```diff
@@ -51,14 +50,14 @@ GetNodeCommand.ShortDescription=Esegue il dump dell''XML della definizione del n
...
 
 BuildCommand.CLICause.ShortDescription=Avviato dalla riga di comando da {0}
-BuildCommand.CLICause.CannotBuildDisabled=Impossibile compilare {0} perch� � disabilitato.
-BuildCommand.CLICause.CannotBuildConfigNotSaved=Impossibile compilare {0} perch� la sua configurazione non � stata salvata.
+BuildCommand.CLICause.CannotBuildDisabled=Impossibile compilare {0} perch� � disabilitato.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
BuildCommand.CLICause.CannotBuildDisabled=Impossibile compilare {0} perché è disabilitato.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 27]
```diff
@@ -51,14 +50,14 @@ GetNodeCommand.ShortDescription=Esegue il dump dell''XML della definizione del n
...
+BuildCommand.CLICause.CannotBuildConfigNotSaved=Impossibile compilare {0} perch� la sua configurazione non � stata salvata.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
BuildCommand.CLICause.CannotBuildConfigNotSaved=Impossibile compilare {0} perché la sua configurazione non è stata salvata.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 28]
```diff
@@ -68,9 +67,9 @@ ClearQueueCommand.ShortDescription=Pulisce la coda di lavoro.
 ReloadConfigurationCommand.ShortDescription=Scarta tutti i dati caricati in memoria e ricarica tutto dal file system. Utile quando si sono modificati file di configurazione direttamente su disco.
 ConnectNodeCommand.ShortDescription=Esegue una nuova connessione al nodo/i.
 DisconnectNodeCommand.ShortDescription=Si disconnette da un nodo/i.
-QuietDownCommand.ShortDescription=Ferma le attivit� di Jenkins in preparazione a un riavvio. Non avvia alcuna build.
+QuietDownCommand.ShortDescription=Ferma le attivit� di Jenkins in preparazione a un riavvio. Non avvia alcuna build.
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
QuietDownCommand.ShortDescription=Ferma le attività di Jenkins in preparazione a un riavvio. Non avvia alcuna build.
```

File: core/src/main/resources/hudson/cli/Messages_it.properties [hunk 29]
```diff
@@ -68,9 +67,9 @@ ClearQueueCommand.ShortDescription=Pulisce la coda di lavoro.
...
 CancelQuietDownCommand.ShortDescription=Annulla gli effetti del comando "quiet-down".
-OfflineNodeCommand.ShortDescription=Non utilizza pi� temporaneamente un nodo per eseguire le build fino al successivo comando "online-node".
+OfflineNodeCommand.ShortDescription=Non utilizza pi� temporaneamente un nodo per eseguire le build fino al successivo comando "online-node".
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/cli/Messages_it.properties]:
```suggestion
OfflineNodeCommand.ShortDescription=Non utilizza più temporaneamente un nodo per eseguire le build fino al successivo comando "online-node".
```

**Comment by oleg-nenashev** [comment]
I fixed broken symbols from the web UI , but it does not loo to be enough apparently

**Comment by oleg-nenashev** [comment]
```
ERROR] Failed to execute goal org.jvnet.hudson.tools:maven-encoding-plugin:1.1:check-encoding (default) on project jenkins-core: Problems detected -> [Help 1]

org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.jvnet.hudson.tools:maven-encoding-plugin:1.1:check-encoding (default) on project jenkins-core: Problems detected

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:213)

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)

    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)

    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)

    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)

    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)

    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)

    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)

    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)

    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)

    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)

    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)

    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)

    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)

    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)

    at java.lang.reflect.Method.invoke (Method.java:566)

    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)

    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)

    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)

    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)

Caused by: org.apache.maven.plugin.MojoExecutionException: Problems detected

    at org.jvnet.hudson.tools.PropertyFileEncodingCheckerMojo.execute (PropertyFileEncodingCheckerMojo.java:78)

    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)

    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)

    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)

    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)

    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)

    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)

    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)

    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)

    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)

    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)

    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)

    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)

    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)

    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)

    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)

    at java.lang.reflect.Method.invoke (Method.java:566)

    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)

    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)

    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)

    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)

[ERROR] 

[ERROR] Re-run Maven using the -X switch to enable full debug logging.

```

:(

**Review by alecharp — CHANGES_REQUESTED** [review]
I see way more changes than it should have.

**Comment by daniel-beck** [comment]
> I see way more changes than it should have.

I don't -- just way more highlighted lines. Is this a file encoding change?

**Comment by oleg-nenashev** [comment]
yes, encoding.
I have no quick way to fix it on my laptop, it seems

**Pull request closed without merging**